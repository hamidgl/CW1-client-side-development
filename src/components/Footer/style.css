footer {
  height: 60px;
  width: 100%;
  background-color: black;
  color: white;
  text-align: center;
  padding-top: 15px;
  margin-top: 15px;
}
