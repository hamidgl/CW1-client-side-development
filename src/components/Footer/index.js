import React from "react";
import "./style.css";

const Footer = () => {
  return (
    <>
      <footer className="container">
        Â© 2021 VisitScotland. All rights reserved.
      </footer>
    </>
  );
};

export default Footer;
