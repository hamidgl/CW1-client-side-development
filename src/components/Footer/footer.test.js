import * as React from "react";
import { render } from "@testing-library/react";
import Footer from "./";

// npm run test (to run the test)
// npm run test -- -u (to update snapshot test failure)

describe("<Footer />", () => {
  test("should render component", () => {
    const result = render(<Footer />);
    expect(result.container).toMatchSnapshot();
  });

  test("should display footer text", () => {
    const { queryByText } = render(<Footer />);
    const footerText = queryByText(
      "Â© 2021 VisitScotland. All rights reserved."
    );

    expect(footerText).toBeInTheDocument();
  });
});
