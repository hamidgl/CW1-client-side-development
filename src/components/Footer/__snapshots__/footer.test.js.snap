// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render component 1`] = `
<div>
  <footer
    class="container"
  >
    Â© 2021 VisitScotland. All rights reserved.
  </footer>
</div>
`;
